1_fetch.R

```diff
@@ -430,6 +430,37 @@ p1_targets_list <- list(
   tar_target(
     p1_reservoirs_sf,
     readRDS(p1_reservoirs_rds)
+  ),
+  
+  # Read in .rds file containing the DRB adjacency matrix, referenced to
+  # subseg_id (PRMS_segid). subseg_distance_matrix.rds was created as part
+  # of the delaware-model-prep pipeline and was downloaded on 11/3/2022.
+  # https://github.com/USGS-R/delaware-model-prep/blob/main/1_network.yml#L69
+  tar_target(
+    p1_drb_distance_matrix_rds,
+    "1_fetch/in/subseg_distance_matrix.rds",
+    format = "file",
+    repository = "local"
+  ),
+  tar_target(
+    p1_drb_distance_matrix,
+    readRDS(p1_drb_distance_matrix_rds)
+  ),
+  tar_target(
+    p1_drb_distance_matrix_updown,
+    p1_drb_distance_matrix$updown
+  ),
+  #save as csv for use in Python
+  tar_target(
+    p1_drb_distance_matrix_updown_csv,
+    {
+      write.csv(x = as.data.frame(p1_drb_distance_matrix$updown), 
+              file = "1_fetch/out/drb_distance_matrix_updown.csv",
+              row.names = TRUE)
+      "1_fetch/out/drb_distance_matrix_updown.csv"
+    },
```

Comment:
Oh, great addition - I wasn't thinking about the need to export for python!

One small comment that writing the file path twice increases the likelihood of making a typo - this looks good as you've written it but here's a proposed edit I think we've used elsewhere in this pipeline:

```suggestion
    {
      fileout <- "1_fetch/out/drb_distance_matrix_updown.csv"
      write.csv(x = as.data.frame(p1_drb_distance_matrix$updown), 
              file = fileout,
              row.names = TRUE)
      fileout
    },
```